Week3/homework/addSix.js

```diff
@@ -0,0 +1,11 @@
+function createBase(a) {
+    return function func1() {
```

Comment:
no need to name `funct1` here. This name is ignored. It may be used only when you want to describe what this function does. eg `increaseValue`